.travis.yml

```diff
@@ -16,17 +16,28 @@ cache: pip
 
 jobs:
   include:
-    - name: "dependencies"
+    - name: "production dependencies"
       script:
         - pip check
+    - name: "development dependencies"
+        script:
+          - pip install -U -r requirements.txt -r requirements-dev.txt .
+          - pip check
     - name: "coverage"
       script:
+        - pip install --upgrade pip
         - pip install -U -r requirements-dev.txt .
         - coverage run --branch --omit="*.html" --source cg setup.py test
         - coveralls
+    - name: "code formatting"
+      script:
+        - pip install --upgrade pip
+        - pip install black
+        - black --check cg tests
     - name: "linting"
       script:
-        - pip install -r requirements-dev.txt
+        - pip install --upgrade pip
```

Comment:
Same here: linting requires the latest version of pip?

Reply:
no